api/envoy/extensions/transport_sockets/s2a/v3alpha/s2a.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package envoy.extensions.transport_sockets.s2a.v3alpha;
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.transport_sockets.s2a.v3alpha";
+option java_outer_classname = "S2aProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Secure Session Agent (S2A)]
+// [#extension: envoy.transport_sockets.s2a]
+
+// Configuration for S2A transport socket. This allows Envoy clients to
```

Comment:
Are there any links to public S2A documents available?

Reply:
Currently, we do not have dedicated public docs for S2A. The closest thing would be [this](https://github.com/google/s2a-core#readme) description. Should we add this for now until we have a more permanent location?

Reply:
Went ahead and removed the import.

Reply:
Yes please!

Reply:
Done. Thanks!